tests/e2e/gateway/02-assert.yaml

```diff
@@ -0,0 +1,212 @@
+apiVersion: v1
+data:
+  rbac.yaml: "tenants:\n- name: test-oidc\n  id: test-oidc\n  oidc:\n    issuerURL: https://dex.klimlive.de/dex\n    \n  opa:\n    query: data.tempo.allow\n    paths:\n \   - /etc/tempo-gateway/cm/rbac.yaml\n    - /etc/tempo-gateway/cm/tempo-gateway.regodc"
```

Comment:
Could you add this as multiline yaml instead of a string with encoded newlines?
https://github.com/kubernetes/kubernetes/issues/36222#issuecomment-1062121115

Reply:
+1